packages/beeq/src/components/date-picker/scss/bq-date-picker.variables.scss

```diff
@@ -48,6 +48,7 @@
   --bq-date-picker--padding-end: theme('spacing.m');
   --bq-date-picker--paddingY: theme('spacing.s');
 
+  --bq-date-picker--day-size: theme(spacing.xl);
```

Comment:
I think that here should be `--bq-date-picker--day-size: theme('spacing.xl');` isn't it?

Reply:
Ok, got it!